packages/js/plugins/ethereum/src/utils/storage.ts

```diff
@@ -0,0 +1,53 @@
+import { Connection } from "@polywrap/ethereum-plugin-js";
```

Comment:
Having this file here is odd, as it's only used for testing. I'd suggest moving it within the `__tests__` directory.

Reply:
my bad! I thought it was in ^^